export-workflow-statistics/config.py

```diff
@@ -0,0 +1,17 @@
+# Configuration
+
+url = "https://opencast.com"
+url_pattern = "https://{}.opencast.com"
```

Comment:
I have set url and url_pattern to the opencast admin url.
This is only neccessarry for non multi tenancy.
If thats correct a hint somewhere would be nice

Reply:
+1. And perhaps defaults to localhost? Having the default point to our main website could be confusing.